data/tutorials/language/0lg_09_env.md

```diff
@@ -0,0 +1,675 @@
+---
+id: values-and-functions
+title: Values and Functions
+description: |
+  Functions, values, definitions, environments, closures and shadowing: this tutorial will help you master the fundamentals.
+category: "Language"
+---
+
+# Values and Functions
+
+## Introduction
+
+This tutorial teaches the skills needed to handle expressions, values, and names. You'll learn the ability to write expressions, name values or leave them anonymous, appropriately scope names, handle multiple definitions of the same name, create and use closures, and produce or avoid side effects.
+
+In OCaml, functions are values. In comparison to other mainstream languages, this creates a richer picture between expressions, values, and names. The approach in this tutorial is to acquire the related capabilities and understanding by interacting with OCaml in UTop. This hands-on experience is intended to build the understanding by experimentation rather than starting with the definition of the concepts.
```

Comment:
```suggestion
In OCaml, functions are values. In comparison to other mainstream languages, this creates a richer picture between expressions, values, and names. The approach in this tutorial is to acquire the related capabilities and understanding by interacting with OCaml in UTop. This hands-on experience is intended to build understanding by experimentation rather than starting with the definition of the concepts.
```